recipes/wsinfer/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "wsinfer" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/wsinfer-{{ version }}.tar.gz
+  sha256: c021c137df10f6206c52a56296ce50ff5dfbb1eb463392dc025a5c59de5af8b8
+
+build:
+  skip: true  # [win]
```

If you are skipping Windows b/c a missing dependency there, this still can be noarch. The package won't be installable on Windows until that dependency exist there, so no harm done.

```suggestion
  noarch: python
```